### ISSUE TITLE
Adding support for additional languages

### ISSUE THREAD

**Issue by johnboc** [Wanderxx/vue-fullcalendar#102]
I can see en, zh, and fr languages in langSets.js. How do I add support for additional languages, such as es. My Vue file is:

 <template>
        <full-calendar :events="events" locale="es"></full-calendar>
</template>

<script>
import Event from '../models/Event';
import fullCalendar from 'vue-fullcalendar';

export default {
    components: {
        fullCalendar,
    },

    data: function() {
        return {
            events: [],
        }
    },

mounted: function() {
        Event.getCalendar(events => this.events = events);
    },

  }

**Comment by Lubmes** [comment]
Or 'nl'

**Comment by tiagosimoesdev** [comment]
Yeah I need pt aswell , it would be great if someone could explain how should we add
